src/index.js

```diff
@@ -0,0 +1,65 @@
+import './style.css';
+
+const addBtn = document.querySelector('#add-btn');
+const todoList = document.querySelector('.todo-list');
+const inputField = document.querySelector('#input-field');
+const clearAll = document.querySelector('.clearAll');
+const container = document.querySelector('.container');
+const inputDiv = document.querySelector('.input-div');
+
+const todoListArray = [
+  {
+    description: 'none',
+    completed: false,
+    index: 0,
+  },
+];
```

Comment:
- [ ] You should have a function that iterates over the tasks array and populate the HTML list item element for each task. Let's add it so that this initial task gets shown on page load.